ui/src/ensembler/components/table/ListEnsemblingJobsForEnsemblerTable.js

```diff
@@ -0,0 +1,120 @@
+import React, { Fragment, useEffect, useState } from "react";
+import {
+  EuiCallOut,
+  EuiBasicTable,
+  EuiText,
+} from "@elastic/eui";
+import { DeploymentStatusHealth } from "../../../components/status_health/DeploymentStatusHealth";
+import { JobStatus } from "../../../services/job/JobStatus";
+import { useConfig } from "../../../config";
+import { useTuringApi } from "../../../hooks/useTuringApi";
+
+
+export const ListEnsemblingJobsForEnsemblerTable = ({
+  projectID,
+  ensemblerID,
+  setCanDeleteEnsembler,
+  canDeleteEnsembler
+}) => {
+  const [results, setResults] = useState({ inactiveItems: [], activeItems:[], totalInactiveCount: 0, totalActiveCount:0 });
+
+  const {
+    appConfig: {
+      tables: { defaultTextSize },
+    },
+  } = useConfig();
+
+  const [{ data, isLoaded, error }] = useTuringApi(
+    `/projects/${projectID}/jobs?ensembler_id=${ensemblerID}`,
+    []
+  )
+
+  useEffect(() => {
+    if (isLoaded && !error) {
+      let inactiveItems = data.results.filter((item) => item.status === 'failed_submission' || item.status==='failed_building' || item.status==='failed' || item.status==='completed');
```

Comment:
May be clearer to do something similar to an `IN` and `NOT IN` comparison for these `inactiveItems` and `activeItems` check. See: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

You've used this function in some other parts of the implementation, you can use it here too.